addons/stock/report/report_stock_forecasted.xml

```diff
@@ -130,9 +137,16 @@
                             <td t-esc="line['receipt_date'] or ''"
                                 t-attf-class="#{line['is_late'] and 'o_grid_warning'}"/>
                             <td t-if="docs['multiple_product']" t-esc="line['product']['display_name']"/>
-                            <td class="text-right"><t t-if="not line['replenishment_filled']">- </t><t t-esc="line['quantity']"/></td>
-                            <td t-esc="line['uom_id'].name" groups="uom.group_uom"/>
-                            <td t-attf-class="#{not line['replenishment_filled'] and 'o_grid_warning'}">
+                            <td class="text-right"><t t-if="not line['replenishment_filled']">- </t><t t-esc="line['quantity']" t-options="{'widget': 'float', 'precision': 2}"/></td>
```

Comment:
'precision': 2 use decimal.precision xml_id decimal_product_uom (Product Unit of Measure)